packages/dds/sequence/src/test/sharedString.spec.ts

```diff
@@ -38,6 +39,10 @@ describe("SharedString", () => {
 
 	beforeEach(() => {
 		dataStoreRuntime1 = new MockFluidDataStoreRuntime();
+		dataStoreRuntime1.options = {
```

Comment:
Rather than enable for all the existing tests in this describe block, can you create a new one?